js/customize-partial-refresh-settings.js

```diff
@@ -0,0 +1,128 @@
+/* global jQuery, _customizePartialRefreshSettings, JSON, _ */
+/* exported customizePartialRefreshSettings */
+var customizePartialRefreshSettings = ( function( $ ) {
+	'use strict';
+
+	var self, api = wp.customize;
+
+	self = {
+		setting: null
+	};
+
+	$.extend( self, _customizePartialRefreshSettings );
+
+	/**
+	 * Inject the functionality.
+	 */
+	self.init = function() {
+		api.bind( 'ready', function() {
+			self.onChangeSetting();
+		} );
+
+		api.bind( 'setting-partial', function( response ) {
+			var that = $( 'iframe' ).contents(),
+				value = $.trim( response );
+
+			$( self.setting.selector, that ).html( value );
+		} );
+	};
+
+	/**
+	 * Change event for selective settings, requests a new value.
+	 */
+	self.onChangeSetting = function() {
+		$.each( self.settings, function( id, selector ) {
+
+			api( id, function( setting ) {
+
+				setting.bind( function() {
+					self.setting = {
+						id: id,
+						selector: selector
+					};
+					self.request();
+				} );
+			} );
+		} );
+	};
+
+	/**
+	 * Request a new setting value.
+	 *
+	 * @return {jQuery.Deferred}
+	 */
+	self.request = function() {
+		var spinner = $( '#customize-header-actions .spinner' ),
+			active = 'is-active',
+			deferred = $.Deferred(),
+			req = self.debounceRequest();
+
+		spinner.addClass( active );
+
+		req.done( function( response ) {
+			deferred.resolve();
+			api.trigger( 'setting-partial', response );
```

Comment:
Interesting. It didn't occur to me to do the Ajax request for the partial in the pane and then send the response into the preview.